src/User32.Desktop/User32+DLGITEMTEMPLATE.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.

namespace PInvoke
{
    using System.Runtime.InteropServices;

    /// <content>
    /// Contains the <see cref="DLGITEMTEMPLATE"/> nested type.
    /// </content>
    public partial class User32
    {
        /// <summary>
+        ///     Defines the dimensions and style of a control in a dialog box. One or more of these structures are combined with a
+        ///     <see cref="DLGTEMPLATE" /> structure to form a standard template for a dialog box.
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential, Pack = 2)]
+        public struct DLGITEMTEMPLATE
+        {
+            /// <summary>
+            ///     The style of the control. This member can be a combination of window style values (such as WS_BORDER) and one or
+            ///     more of the control style values (such as BS_PUSHBUTTON and ES_LEFT).
+            /// </summary>
+            public int style;
+
+            /// <summary>
+            ///     The extended styles for a window. This member is not used to create controls in dialog boxes, but applications that
+            ///     use dialog box templates can use it to create other types of windows.
+            /// </summary>

```

Comment:
Wow. What was your clue that you needed to set `Pack = 2`? I'm suddenly worried that if this is important, we may have missed it in other structures.